.github/workflows/docker-cli-image.yaml

```diff
@@ -41,12 +41,20 @@ jobs:
 
     steps:
       - name: Delete huge unnecessary tools folder
-        run: rm -rf /opt/hostedtoolcache
+        run: |
+          rm -rf /opt/hostedtoolcache
+          rm -rf /usr/share/dotnet
+          rm -rf /opt/ghc
+          rm -rf "/usr/local/share/boost"
+          rm -rf "$AGENT_TOOLSDIRECTORY"
       - uses: actions/checkout@v3
       - uses: actions/setup-java@v3
         with:
           java-version: '17'
           distribution: 'corretto'
+      - uses: dtolnay/rust-toolchain@1.79.0
+      - name: Install cargo cross
+        run: cargo install cross
```

Comment:
Is this needed if we set the skipRust property for the environment image Maven build? It's very time consuming to compile the Rust code.